src/helpers/userHelper.js

```diff
@@ -692,75 +713,83 @@ const userHelper = function () {
     newInfringements = _.differenceWith(
       newCurrent,
       newOriginal,
-      (arrVal, othVal) => arrVal._id.equals(othVal._id),
+      (arrVal, othVal) => arrVal._id.equals(othVal._id)
     );
     newInfringements.forEach((element) => {
       emailSender(
         emailAddress,
-        'New Infringement Assigned',
+        "New Infringement Assigned",
         getInfringementEmailBody(
           firstName,
           lastName,
           element,
-          totalInfringements,
+          totalInfringements
         ),
         null,
-        'onecommunityglobal@gmail.com',
+        "onecommunityglobal@gmail.com"
       );
     });
   };
 
   const replaceBadge = async function (personId, oldBadgeId, newBadgeId) {
     userProfile.updateOne(
-      { _id: personId, 'badgeCollection.badge': oldBadgeId },
+      { _id: personId, "badgeCollection.badge": oldBadgeId },
       {
         $set: {
-          'badgeCollection.$.badge': newBadgeId,
-          'badgeCollection.$.lastModified': Date.now().toString(),
-          'badgeCollection.$.count': 1,
+          "badgeCollection.$.badge": newBadgeId,
+          "badgeCollection.$.lastModified": Date.now().toString(),
+          "badgeCollection.$.count": 1,
         },
       },
       (err) => {
         if (err) {
           console.log(err);
         }
-      },
+      }
     );
   };
 
   const increaseBadgeCount = async function (personId, badgeId) {
-    console.log('Increase Badge Count', personId, badgeId);
-    userProfile.updateOne({ _id: personId, 'badgeCollection.badge': badgeId },
-    { $inc: { 'badgeCollection.$.count': 1 }, $set: { 'badgeCollection.$.lastModified': Date.now().toString() } , $push:{'badgeCollection.$.earnedDate':earnedDateBadge() }},
-    (err) => {
-      if (err) {
-        console.log(err);
+    console.log("Increase Badge Count", personId, badgeId);
```

Comment:
As chen as stated, please remove console.logs used for testing because they hinder performance.